validator/validator-main.protoascii

```diff
@@ -5321,6 +5320,39 @@ tags: {
   }
 }
 
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "IMG"
+  spec_name: 'I-AMPHTML-INTRINSIC-SIZER'
+  mandatory_parent: "I-AMPHTML-SIZER"
+  attrs: {
+    name: "class"
+    value_regex: "i-amphtml-intrinsic-sizer"
+    mandatory: true
+  }
+  attrs: {
+    name: "role"
+    value: "presentation"
+    mandatory: true
+  }
+  attrs: {
+    name: "alt"
+    value: ""
+    mandatory: true
+  }
+  attrs: {
+    name: "aria-hidden"
+    value: "true"
+    mandatory: true
+  }
+  attrs: {
+    name: "src"
+    value_regex: "^data:image\/svg\+xml;charset=utf-8,<svg.*"
```

Comment:
Can we make this even more precise by only allowing variants in the `width=` and `height=` values inside the SVG?

Reply:
Is it OK to enforce a specific order in the attributes?

Reply:
Probably not great if this is publisher generated, but transformers should be predictable enough, unless you expect someone to make a custom implementation. Even then, it should be fairly easy for them to validate against this.

Reply:
done